centreon/www/class/config-generate/generate.class.php

```diff
@@ -328,6 +338,14 @@ private function configPoller($username = 'unknown'): void
             Backend::getInstance($this->dependencyInjector),
             $readAdditionalConnectorRepository
         ))->generateFromPollerId($this->current_poller['id']);
+        $readAgentConfigurationRepository = $kernel->getContainer()->get(
+            ReadAgentConfigurationRepositoryInterface::class
```

Comment:
if you need this class, you can pass it in the constructor with a dependencie injection

Reply:
ok